int64 stop
---
string stopped